base.Dockerfile

```diff
@@ -121,10 +117,12 @@ ENV PATH=${TMPDIR_FOR_RUBY_WRAPPERS_DIR}:${PATH}
 
 # Install node.js, yarn and other runtime dependencies.
 COPY --from=builder /usr/share/keyrings/nodesource.gpg /usr/share/keyrings/
+
+# hadolint ignore=DL3015
 RUN install_packages ca-certificates curl libjemalloc-dev libgdbm6 libyaml-0-2 \
       libmariadb3 libpq5 mariadb-client postgresql-client tzdata; \
-    echo "deb [signed-by=/usr/share/keyrings/nodesource.gpg] https://deb.nodesource.com/node_18.x jammy main" | tee /etc/apt/sources.list.d/nodesource.list; \
-    install_packages nodejs; \
+      curl -fsSL https://deb.nodesource.com/setup_18.x | bash; \
+    apt-get install -y nodejs; \
```

Comment:
Can you specify `--no-install-recommends` and remove the `ignore=DL3015`? If there are other packages that are needed then we should specify them explicitly.